README.md

```diff
@@ -2,9 +2,18 @@
 
 This is a collection of tools designed to consume a compiled Apple Swift library and generate bindings and wrappers that enable it to be used as a .NET library.
 
-## Current Status
-
-The components are currently undergoing iterative reviews and will be moved from https://github.com/xamarin/binding-tools-for-swift.
+## Installation and usage
+
+The tool will be available as a NuGet CLI package in the [`dotnet-experimental`](https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-experimental) feed.
+
+Options:
+```
+  -d, --dylibs             Required. Paths to the dynamic libraries (dylibs), separated by commas.
```

Comment:
The original design was to operate on multiple libraries, but frankly, this made things worse overall. I feel like we should simplify and make it take only a single dylib/swiftinterface pair.